src/toolbars/ControlToolBar.cpp

```diff
@@ -150,33 +150,6 @@ void ControlToolBar::Create(wxWindow * parent)
    UpdatePrefs();
 }
 
-// This is a convenience function that allows for button creation in
-// MakeButtons() with fewer arguments
-AButton *ControlToolBar::MakeButton(ControlToolBar *pBar,
```

Comment:
A separate commit just to move this function into Toolbar.cpp might be nice too.

Reply:
Actually though... you don't really use this overload in the final commit